crates/bevy_sprite/src/texture_slice/computed_slices.rs

```diff
@@ -120,8 +121,8 @@ fn compute_sprite_slices(
             };
             slice.tiled(*stretch_value, (*tile_x, *tile_y))
         }
-        SpriteImageMode::Auto => {
-            unreachable!("Slices should not be computed for SpriteImageMode::Stretch")
+        SpriteImageMode::Auto | SpriteImageMode::ScaleMode(_) => {
```

Comment:
This will be clearer for debugging if you keep them as two different match arms. Then you'll get a more precise warning if this ever breaks.